includes/Hooks.php

```diff
@@ -96,16 +97,39 @@ public function onSpecialPageBeforeExecute( $special, $subPage ) {
 			return true;
 		}
 
+		$config = MediaWikiServices::getInstance()->getMainConfig();
+		$protectedSpecialPages = $config->get( 'CrawlerProtectedSpecialPages' );
+		$denyFast = $config->get( 'CrawlerProtectedSpecialPages' );
+
 		$name = strtolower( $special->getName() );
-		if ( in_array( $name, [ 'recentchangeslinked', 'whatlinkshere' ], true ) ) {
+		if (
+			// allow forgiving entries in the setting array for Special pages names
+			in_array( $special->getName(), $protectedSpecialPages, true )
+			|| in_array( $name, $protectedSpecialPages, true )
+			|| in_array( 'Special:' . $special->getName(), $protectedSpecialPages, true )
```

Comment:
Please refactor magic word `"Special:"` to a constant variable at the top of the file